tests/testthat/test-layers-list.R

```diff
@@ -0,0 +1,25 @@
+context("layers")
+
+# Disable sass cache
+local_disable_cache()
+
+body_rule <- "body { background-color: $color; color: my_invert($color); }"
+
+blue <- list(color = "blue !default")
+red <- list(color = "red !default")
+green <- list(color = "green !default")
+core <- sass_layer(
+  defaults = blue,
+  declarations = "@function my_invert($color, $amount: 100%) {
+    $inverse: change-color($color, $hue: hue($color) + 180);
+    @return mix($inverse, $color, $amount);
+  }",
```

Comment:
Seems the declarations here is unnecessary (and makes the test harder to read)?

```suggestion
```

Reply:
I'm going to keep it as it changes the input color to `yellow`.  This means that the defaults must be ordered properly before the declarations.